databroker/assets/core.py

```diff
@@ -168,7 +168,38 @@ class _PrivateException(Exception):
 def insert_resource(col, spec, resource_path, resource_kwargs,
                     known_spec, root, path_semantics='posix', uid=None,
                     run_start=None,
+                    id=None,
                     ignore_duplicate_error=False, duplicate_exc=None):
+    """Insert resource into a databroker
```

Comment:
To make it a complete sentence:
```suggestion
    """Insert resource into a databroker.
```